docs/user/exams/instructors_guide.rst

```diff
@@ -0,0 +1,192 @@
+
+.. figure:: instructor/programming_assessment.png
+   :alt: Programming Submission Assessment
+   :align: center
+
+   Manually Assessing a Programming Submission
+
+- Quiz exercises are graded automatically via the student exam page, see `Manage Student Exams`_, and therefore do not appear in the *Tutor Exam Dashboard*.
+- Students can complain about their results during the :ref:`student review <student_review>` period. You can evaluate these complaints also in the *Tutor Exam Dashboard*. 
+- The original assessor may not respond to the complaint about his assessment, this must be done by a second assessor. 
+
+Exam Scores
+^^^^^^^^^^^
+- You can view the exam scores from the *Scores* page. This view aggregates the results of the students and combines them to provide an overview over the students’ performance. 
+- You can view the spread between different achieved scores, the average results per exercise as well as the individual students' results.
+- Additionally, you can choose to modify the dataset by selecting ``only include submitted exams`` or ``only include exercises with at least one non-empty submission``.
+- The exam scores can also be exported via ``Export Results as CSV``.
+
+.. figure:: instructor/exam_statistics.png
+   :alt: Exam Scores page
+   :align: center
+
+   Exam Scores Page
+
+Conducting Test Runs
+^^^^^^^^^^^^^^^^^^^^
+- Test runs are designed to offer the instructors confidence that the :ref:`exam conduction <exam_conduction>` will run smoothly. They allow you to experience the exam from the student’s perspective. A ``test run`` is distinct from a ``student exam`` and is not taken into consideration during the calculation of the exam scores. 
+- You can manage your test runs from the *Test Run* page.
+- To create a new test run you can press ``Create a Test Run``. This will open a modal where you can select an exercise for each exercise group. You can also set the ``working time``. A test run will have as many exercises, as there are exercise groups. It does not take the ``number of exercises`` set in the exam configuration under consideration.
+
+    - **Hint:** Exercise groups with no exercises are ignored.
+
+- When you start the test run, you conduct the exam similar to how a student would. You can create submissions for the different exercises and end the test run. 
+- An instructor can also assess his test run submissions. To do this, you must have completed at least one test run. To navigate to the assessment screen of the test runs click ``Assess your Test Runs``. 
+
+    - **Hint:** Only the creator of the test run is able to assess his submissions.
+
+- You can view the results of the assessment of the test run by clicking on ``Summary``. This page simulates the *Student Exam Summary* where the students can view their submissions and the results once they are published. 
+- Here instructors can also use the ``complaint`` feature and respond to it to conclude the full  :ref:`exam timeline <timeline>`.
```

Comment:
I would add a hint that test runs should be delete before the actual exam conduction takes place